src/smart-components/group/add-group/group-information.js

```diff
@@ -18,7 +18,9 @@ const GroupInformation = (formValue, onHandleChange, setIsGroupInfoValid, isGrou
             fieldId="group-name"
             helperTextInvalid={
               formValue.name?.trim().length > 0 && !isValidating
-                ? 'This group name already exists. Please input a unique group name.'
+                ? formValue.name?.trim().length < 150
```

Comment:
I don't think we should trim the length.  I think you can crash the API even with 150 spaces. Try sending something like 
```
a                                                      b                                            c                               #actual end of the string
```